pyscf/pbc/df/test/test_rsdf.py

```diff
@@ -0,0 +1,80 @@
+# Copyright 2014-2018 The PySCF Developers. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+import numpy
+from pyscf import lib
+import pyscf.pbc
+from pyscf import ao2mo, gto
+from pyscf.pbc import gto as pgto
+from pyscf.pbc import scf as pscf
+from pyscf.pbc.df import rsdf
+
+cell = pgto.Cell(
+    atom="H 0 0 0; H 0.75 0 0",
+    a = numpy.eye(3)*3,
+    basis={"H": [[0,(0.5,1.)],[1,(0.3,1.)]]},
+)
+cell.verbose = 0
+cell.max_memory = 1000
+cell.build()
+scaled_center = numpy.array([0.392, 0.105, 0.872])
+
+
+def tearDownModule():
+    global cell
+    del cell
+
+class KnownValues(unittest.TestCase):
+    def test_h2_gamma(self):
+        mf = pscf.KRHF(cell).rs_density_fit()
+        mf.kernel()
+        self.assertAlmostEqual(mf.e_tot, -1.0430635251, 6)
```

Comment:
Is `6` the accuracy you'd expect with the default RSGDF settings?

Reply:
Changed to 7. RSDF computes integrals to be `cell.precision` which by default is 1e-8. It's not clear exactly how that is translated to the precision in SCF energy but a bit larger than that is expected in general.